source/_patterns/00-atoms/02-text/_00-typography.scss

```diff
@@ -0,0 +1,157 @@
+
+
+/* text-size proportional spacer */
+:root {
+  --body-text-spacer: clamp(1rem, 2vw, 1.25rem);
+}
+
+:root {
+  --content-width: min(93vw, 44rem);
+  --margin-left: calc( (100vw - var(--content-width)) / 2 );
+  --margin-right: calc( (100vw - var(--content-width)) / 2 );
+}
+
+:root {
+  --font-sans-fallback: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  --font-serif-fallback: Georgia, 'Times New Roman', serif;
+  --font-sans-static: "Work Sans", var(--font-sans-fallback);
+  --font-sans-vf: "Work Sans VF", var(--font-sans-fallback);
+  --font-serif-static: "Roboto Slab", var(--font-serif-fallback);
+  --font-serif-vf: "Roboto Slab VF", var(--font-serif-fallback);
+}
+
+
+p {
+  font-family: var(--font-sans-vf);
+  font-size: clamp(1rem, 2vw, 1.25rem);
+	font-weight: 350;
+  font-style: normal;
+  hyphens: auto;
+  line-height: clamp(1.25em, 2vw, 1.4em);
+  margin: 0 var(--margin-right) 0 var(--margin-left);
+  max-width: var(--content-width);
+  @media screen and (min-width: $bp--medium) {
+    hyphens: none;
+  }
+  + p,
+  + ul {
+    margin-top: var(--body-text-spacer);
+  }
+}
+
+h1 {
+  color: $color-quahog;
+  font-family: var(--font-serif-vf);
+  font-size: clamp(2rem, 5vw, 5.5rem);
+	font-weight: 275;
+  font-style: normal;
+  line-height: clamp(1.05em, 6vw, 1.1em);
+  margin-top: calc( var(--body-text-spacer) * 2 );
+  margin-left: var(--margin-left);
+  margin-right: calc( var(--margin-right) / 2 );
```

Comment:
What is the purpose of margin-left and margin-right on all of these headings? If I'm looking at the basic page mockup the heading are left aligned inside a section that will already have its width derived. This makes it seem like they will then get additional margin.

Reply:
Removed - that was left over from the test page layout